.github/scripts/pr_mod_file_tests.py

```diff
@@ -197,10 +197,15 @@ def _main_prog():
         #PR itself, so don't check its file type:
         if os.path.exists(file_obj.filename):
 
-            #Check if it is a python file:
-            if _file_is_python(file_obj.filename):
-                #If so, then add to python list:
-                pyfiles.append(file_obj.filename)
+            #Don't analyze 'manage_externals' files,
+            #as they are an external repo and thus
+            #not our responsibility:
+            if not 'manage_externals' in file_obj.filename:
```

Comment:
Interesting. [PEP 8](https://peps.python.org/pep-0008/) mentions `_ not in _` but not `not _ in _`.

Reply:
If I'm reading this correctly, `not in` looks to be an explicit comparison operator but `not _ in _` might be treaded as a comparison operator wrapped with a boolean operator?

Reply:
I agree with those interpretations. Of course, since `in` has a higher precedence than `not`, the logic table is the same. Depends on who gets to the [style guide] (https://github.com/ESCOMP/CAM-SIMA/wiki/CAM%E2%80%90SIMA-Coding-Standards#python-coding-standards) first ;)?
@nusbaume, any thoughts?

Reply:
Well all else being equal in English I think (?) we usually say "x not in y", and I was originally attending the behavior of the `not in` interpretation, so I've gone ahead and changed it.

Also no worries about being pedantic!  If there is one place it's called for it's probably in the testing code where it is checking if everyone else is being properly pedantic.